app/views/projects/show.html.erb

```diff
@@ -15,7 +15,11 @@
         <th class="project-table__cell">Best<br />Estimate</th>
         <th class="project-table__cell">Worst<br />Estimate</th>
         <th class="project-table__cell story_actions">
-          <%= link_to "Add a Story", new_project_story_path(@project.id), class: "button magenta"  %>
+          <% if @project.archived? %>
```

Comment:
I wonder if there's a way to remove this repetition of `if > button > else > link`, maybe move this to a helper method? there's something about it that makes me feel it will be hard to maintain.

Options I can imagine:
- add the aria-disabled and disabled attributes to the actual element (the disabled attribute will prevent the click anyway)
- move this if/else into an application helper that will hide this implementation detail, something like the link_to_if and link_to_unless helpers https://apidock.com/rails/v6.1.3.1/ActionView/Helpers/UrlHelper/link_to_if but with a custom fallback behavior

we can skip adding the `disabled` class too, we can target this in css with the `:disabled` pseudo-class instead of `.disabled`